XCUITests/AuthenticationTest.swift

```diff
@@ -60,6 +60,7 @@ class AuthenticationTest: BaseTestCase {
     }
     
     fileprivate func enablePasscode(_ passCode: String) {
+        navigator.goto(NewTabMenu)
```

Comment:
As No-Jun commented that is solved with gesture instead of tap. Would that be a correct solution? Thanks!